Resources/Maps/_Mono/Shuttles/Nfsd/anura.yml

```
@@ -542,7 +542,20 @@ entities:
     - type: RadiationGridResistance
     - type: BecomesStation
       id: Anura
-  - uid: 333
+  - uid: 2
+    components:
+    - type: MetaData
+    - type: Transform
+      pos: 0.5008907,-4.900281
+      parent: 1
+    - type: RadarBlip
+      enabled: False
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)